dns/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Copyright 2015 The Kubernetes Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM debian:jessie
```

Comment:
My feeling is that the 100MB we save on disk isn't worth the pain of not having diagnostic tools available when something goes wrong.  Also most of these files should be shared with other images, so one day I'm sure we'll have a more efficient on-disk representation.

I don't know that the image is used at all in normal operation - I copied the makefile from the nginx controller, and I think it's doing static linking.  So it's only for troubleshooting.

That said, I don't think this service will have particularly complicated diagnostic needs (curl, ls and cat should be all we would want) so we could go with `busybox`, on the grounds that that is what e.g. kube-controller-manager uses, which feels similar.